### ISSUE TITLE
Metrics dashboard: chart doesn't have enough unique colours, some colours don't meet accessibility contrast

### ISSUE THREAD

**Issue by jmealingcds** [cds-snc/ai-answers#955]
<img width="1208" height="357" alt="Image" src="https://github.com/user-attachments/assets/264e5baf-77f7-4534-9186-ba371bbe72bb" />

**Comment by jmealingcds** [comment]
The chart issue seems to be part of https://github.com/cds-snc/ai-answers/issues/936

I'll look more into the titles tomorrow (Helpful, (Yes) on the FR page.

**Comment by jmealingcds** [comment]
https://github.com/cds-snc/ai-answers/pull/954
